### ISSUE TITLE
add a create validator task

### ISSUE THREAD

**Issue by uditvira** [hyphacoop/cosmos-ansible#2]
For the devnets we should have a way to automatically create validators and insert them into the genesis files. I've got a local version working like this by modifying `cosmos-ansible/roles/gaia/tasks/main.yml`

additionally we should a configurable chain-id variable. we don't need it when we have specified genesis files, but need it for  quick devnets.
```
# Initialize gaiad home
- name: initialize gaiad
  when: gaiad_home_autoclear or not gaiad_home_exists.stat.exists
  # TODO: should we overwrite?
  shell: |
    PATH=$PATH:/usr/local/go/bin:{{ ansible_env.HOME }}/go/bin
    gaiad init {{inventory_hostname}} --home {{gaiad_home}} --chain-id "theta-devnet"

- name: create validator
  when: gaiad_create_validator
  shell: |
    PATH=$PATH:/usr/local/go/bin:{{ ansible_env.HOME }}/go/bin
    gaiad keys add validator --keyring-backend="test" --home {{gaiad_home}}
    gaiad add-genesis-account validator '10000000000stake' --home {{gaiad_home}} --keyring-backend="test" 
    gaiad gentx validator 5000000000stake --keyring-backend="test" --home {{gaiad_home}} --chain-id "theta-devnet"
    gaiad collect-gentxs --home {{gaiad_home}}
```

**Comment by RangerMauve** [comment]
Am I correct in saying that the genesis file that results from these transactions would need to be copied to other machines? Or is this something that will get appended to the chain?

**Comment by uditvira** [comment]
You're right, this genesis file would be needed on other machines

**Comment by uditvira** [comment]
one way to do this is to create the genesis file locally first and then use ansible to deploy the network

**Comment by RangerMauve** [comment]
Should this be closed by #7 ?

**Comment by uditvira** [comment]
Yes this can be closed!
